src/components/DatePickerRange.react.js

```diff
@@ -169,8 +168,8 @@ export default class DatePickerRange extends Component {
                         with_full_screen_portal && verticalFlag
                     }
                     withPortal={with_portal && verticalFlag}
-                    startDateId={start_date_id}
-                    endDateId={end_date_id}
+                    startDateId={this.state.start_date_id}
+                    endDateId={this.state.end_date_id}
```

Comment:
I think we need to do `props.start_date_id || this.state.start_date_id` here rather than in the constructor - just in case a user decides to change `props.start_date_id` later on. No idea why someone would do that, but if they did it would not make it to the `<DateRangePicker>` this way.